<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tool</class>
 <widget class="QMainWindow" name="Tool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tool</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Nhập code bộ môn</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtName">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(255, 181, 89);
padding: 5px;
height: 30px;
text-transform: uppercase;</string>
        </property>
        <property name="placeholderText">
         <string>Nhập code bộ môn...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_message">
        <property name="styleSheet">
         <string notr="true">color: rgb(112, 89, 5);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nội dung buôi 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="_1">
          <property name="styleSheet">
           <string notr="true">border: 1px solid rgb(255, 181, 89);
padding: 5px;
height: 30px;</string>
          </property>
          <property name="plainText">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Buổi 1...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nội dung buôi 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="_2">
          <property name="styleSheet">
           <string notr="true">border: 1px solid rgb(255, 181, 89);
padding: 5px;
height: 30px;</string>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_18">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nội dung buôi 3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="_3">
          <property name="styleSheet">
           <string notr="true">border: 1px solid rgb(255, 181, 89);
padding: 5px;
height: 30px;</string>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_19">
        <item>
         <widget class="QLabel" name="label_19">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nội dung buôi 4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="_4">
          <property name="styleSheet">
           <string notr="true">border: 1px solid rgb(255, 181, 89);
padding: 5px;
height: 30px;</string>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nội dung buôi 5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="_5">
          <property name="styleSheet">
           <string notr="true">border: 1px solid rgb(255, 181, 89);
padding: 5px;
height: 30px;</string>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nội dung buôi 6</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="_6">
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid rgb(255, 181, 89);
padding: 5px;
height: 30px;</string>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nội dung buôi 7</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="_7">
          <property name="styleSheet">
           <string notr="true">border: 1px solid rgb(255, 181, 89);
padding: 5px;
height: 30px;</string>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_17">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nội dung buôi 8</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="_8">
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid rgb(255, 181, 89);
padding: 5px;
height: 30px;</string>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_22">
        <item>
         <widget class="QLabel" name="label_22">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nội dung buôi 9</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="_9">
          <property name="styleSheet">
           <string notr="true">border: 1px solid rgb(255, 181, 89);
padding: 5px;
height: 30px;</string>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_23">
        <item>
         <widget class="QLabel" name="label_23">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nội dung buôi 10</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="_10">
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid rgb(255, 181, 89);
padding: 5px;
height: 30px;</string>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_20">
        <item>
         <widget class="QLabel" name="label_20">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nội dung buôi 11</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="_11">
          <property name="styleSheet">
           <string notr="true">border: 1px solid rgb(255, 181, 89);
padding: 5px;
height: 30px;</string>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_21">
        <item>
         <widget class="QLabel" name="label_21">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nội dung buôi 12</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="_12">
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid rgb(255, 181, 89);
padding: 5px;
height: 30px;</string>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_25">
        <item>
         <widget class="QLabel" name="label_25">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nội dung buôi 13</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="_13">
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid rgb(255, 181, 89);
padding: 5px;
height: 30px;</string>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_24">
        <item>
         <widget class="QLabel" name="label_24">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nội dung buôi 14</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="_14">
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid rgb(255, 181, 89);
padding: 5px;
height: 30px;</string>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="styleSheet">
       <string notr="true">background: rgb(18, 137, 188);
border:none;
color: #fff;
height: 35px;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>Thêm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
